### PR TITLE
Improve DNS resolving rate

### PR DESCRIPTION
**Pull request by AmaliMatharaarachchi**
### Issues

Fixes https://github.com/wso2/product-microgateway/issues/2699

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

**Review by pubudu538 — APPROVED** [review]

**Review by praminda — APPROVED** [review]

File: resources/conf/config.toml.template [hunk 1]
```diff
@@ -120,6 +120,11 @@ artifactsDirectory = "/home/wso2/artifacts"
   # Disable SSL verification
   disableSslVerification = false
 
+# configs for upstream clusters 
+[router.upstream]
```
Comment (suksw) [inline resources/conf/config.toml.template:124]:
```suggestion
```
Since we have this in line 105

Comment (AmaliMatharaarachchi) [reply resources/conf/config.toml.template]: fixed

**Review by VirajSalaka — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.33

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.33

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.33

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.33

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.33

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.33

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.34

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.34

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.34

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.34

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.34

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.34

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.34

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.34

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.34

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.34

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.34

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment-v2 : 20220223.34

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220228.1

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220228.1

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220228.1

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220228.1

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220228.1

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220228.1

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220228.1

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220228.1

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220228.1

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220228.1

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220228.1

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment-v2 : 20220228.1

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE1 cluster : stage-deployment-v2 : 20220301.3

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : stage-deployment-v2 : 20220301.3

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : prod-deployment-v2 : 20220302.3

**Pull request merged**